docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,382 @@
+Object Detection model
+======================
+
+This tutorial reveals end-to-end solution from installation to model export and optimization for object detection task on a certain example.
+On this page we show how to train, validate, export and optimize ATSS model on WGISD public dataset.
+
+.. note::
+
+  To learn how to deploy the trained model, refer to: :doc:`../deploy`.
+
+  To learn how to run the demo and visualize results, refer to: :doc:`../demo`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- CUDA Toolkit 11.4 
+
+
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick_start guide <../../../get_started/quick_start>` to create a universal virtual environment for all tasks. On the other hand, you can create a task-specific environment following the process below.
```

Comment:
Is the below figure what we expected?
![image](https://user-images.githubusercontent.com/80735344/213085560-9103cc0c-6f09-498e-b742-41d06ff2a521.png)

Reply:
Yes, it's just GitHub can't render cross-pages references in RST properly :)